src/network/networkd-ndisc.c

```diff
@@ -39,10 +39,8 @@ static int ndisc_netlink_handler(sd_netlink *rtnl, sd_netlink_message *m, void *
         link->ndisc_messages--;
 
         r = sd_netlink_message_get_errno(m);
-        if (r < 0 && r != -EEXIST) {
+        if (r < 0 && r != -EEXIST)
                 log_link_error_errno(link, r, "Could not set NDisc route or address: %m");
```

Comment:
hmm, but why does this fail in the first place?

Reply:
It seems a multiple reasons here prefix change one of them .

Reply:
Also I think we should not bring down the dhcp client too when some error happen. The admin could do a wrong conf and in that case whole network will go down where networkd is running. Then admin has to go each of the machines and start manually. I think we should log and keep on running the network clients.